feature-libs/commerce-quotes/core/facade/commerce-quotes.service.ts

```diff
@@ -37,11 +39,25 @@ export class CommerceQuotesService implements CommerceQuotesFacade {
       { reloadOn: [CommerceQuotesListReloadQueryEvent] }
     );
 
+  protected quoteDetailsState$: Query<Quote, unknown[]> =
+    this.queryService.create<Quote>(() =>
+      this.routingService.getRouterState().pipe(
+        withLatestFrom(this.userIdService.takeUserId()),
+        switchMap(([{ state }, userId]) => {
+          return this.commerceQuotesConnector.getQuote(
+            userId,
+            state.params.quoteId
+          );
+        })
```

Comment:
I guess we can remove curly brackets and make it fully arrow function. Just readability purpose.

Reply:
Yeah, I had to add `console.log` inside this operator and I forgot to revert previous function shape. Thanks!